From 31329bc48c38c641252c635dbb6237d089fa8729 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:10:28 -0400
Subject: [powerpc] Convert last uses of __FUNCTION__ to __func__

Message-id: <1410545655-205645-400-git-send-email-dzickus@redhat.com>
Patchwork-id: 94553
O-Subject: [RHEL7 PATCH 399/626] powerpc: Convert last uses of __FUNCTION__ to __func__
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit aba6f4f2e64f9b43aaba6d46c67e9e25a55f4614
Author: Joe Perches <joe@perches.com>
Date:   Tue Mar 25 12:35:03 2014 -0700

    powerpc: Convert last uses of __FUNCTION__ to __func__

    Just about all of these have been converted to __func__,
    so convert the last uses.

    Signed-off-by: Joe Perches <joe@perches.com>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/platforms/pseries/nvram.c b/arch/powerpc/platforms/pseries/nvram.c
index 2f91a9d..9f1dbd5 100644
--- a/arch/powerpc/platforms/pseries/nvram.c
+++ b/arch/powerpc/platforms/pseries/nvram.c
@@ -298,13 +298,13 @@ int nvram_write_os_partition(struct nvram_os_partition *part, char * buff,
 
  rc = ppc_md.nvram_write((char *)&info, sizeof(struct err_log_info), &tmp_index);
  if (rc <= 0) {
-  pr_err("%s: Failed nvram_write (%d)\n", __FUNCTION__, rc);
+  pr_err("%s: Failed nvram_write (%d)\n", __func__, rc);
   return rc;
  }
 
  rc = ppc_md.nvram_write(buff, length, &tmp_index);
  if (rc <= 0) {
-  pr_err("%s: Failed nvram_write (%d)\n", __FUNCTION__, rc);
+  pr_err("%s: Failed nvram_write (%d)\n", __func__, rc);
   return rc;
  }
  
@@ -351,15 +351,14 @@ int nvram_read_partition(struct nvram_os_partition *part, char *buff,
      sizeof(struct err_log_info),
      &tmp_index);
   if (rc <= 0) {
-   pr_err("%s: Failed nvram_read (%d)\n", __FUNCTION__,
-         rc);
+   pr_err("%s: Failed nvram_read (%d)\n", __func__, rc);
    return rc;
   }
  }
 
  rc = ppc_md.nvram_read(buff, length, &tmp_index);
  if (rc <= 0) {
-  pr_err("%s: Failed nvram_read (%d)\n", __FUNCTION__, rc);
+  pr_err("%s: Failed nvram_read (%d)\n", __func__, rc);
   return rc;
  }
 
@@ -869,7 +868,7 @@ static void oops_to_nvram(struct kmsg_dumper *dumper,
   break;
  default:
   pr_err("%s: ignoring unrecognized KMSG_DUMP_* reason %d\n",
-      __FUNCTION__, (int) reason);
+         __func__, (int) reason);
   return;
  }
 
-- 
1.7.1